server/routes/visitsUtils.test.ts

```diff
@@ -1,24 +1,4 @@
-import { format } from 'date-fns'
-import { getParsedDateFromQueryString, getDateTabs, getSlotsSideMenuData } from './visitsUtils'
-
-describe('getParsedDateFromQueryString', () => {
```

Comment:
Re-using this function from 'View visits by date' page, so just moved it to utils function less tied to a specific page. (We have a lot of 'utils' functions in various places - perhaps something to rationalise at some point...🧐)